### PR TITLE
Fix trtl CLI bugs

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

This PR is required for SC-9227. @masskoder created an account on production but asked me to delete it. It is straight forward to delete his user from Auth0 via their UI, but this leaves a disconnected `organization` in the trtl database that used to be associated with the user. In an effort to keep the BFF database clean, I wanted to delete the organization associated with the user account. Unfortunately, the `trtl` CLI program had some bugs in it that did not allow me to do this easily - this PR fixes those bugs. 

The primary issue is that the keys for the `organizations` are the UUID bytes (not a string) so you can't specify the key on the command line to delete it. The `trtl` command had utilities for base64 encoded keys, but they were not consistent (e.g. not on the `db:put` and `db:del` commands) and they were incorrectly named in a couple of cases so they were not used (e.g in the `db:list` command). This PR unifies the functionality and repairs the misnamed variables. It also adds a utility function `uuid-key` to get the base64 encoded uuid as the key for the organization.

I was able to successfully use this new command to list the organizations and delete the correct organization from the `gds-0` pod. 

### Type of change

- [x] bug fix
- [ ] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

You should be able to list the keys as base64 encoded strings:

```
$ trtl db:list -b -n organizations
```

You should also be able to get the data with a base64 encoded string:

```
$ trtl db:get -b -n organizations abcd1234
```

Finally, you should be able to convert a UUID into a base64 encoded string:

```
$ trtl uuid 33a7bd79-50d9-4136-9dcd-0f46024d3c73
```

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ]  ~I have updated the dependencies list~
- [ ]  ~I have recompiled and included new protocol buffers to reflect changes I made~
- [ ]  ~I have added new test fixtures as needed to support added tests~
- [x]   Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)
- [x]  I have moved the associated Shortcut story to "Ready for Review"

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] The CLI program commands make sense and work as expected.

### CODE REVIEW

File: cmd/trtl/main.go [hunk 1]
```diff
@@ -603,23 +637,29 @@ func dbDelete(c *cli.Context) (err error) {
...
 		Options: &pb.Options{
 			ReturnMeta: c.Bool("meta"),
 		},
 	}
+
+	if req.Key, err = wire.DecodeKey(c.String("key"), c.Bool("b64")); err != nil {
+		return cli.Exit(fmt.Errorf("could not decode key: %s", err), 1)
+	}
+
+	// Execute the Put request
```
Comment (pdeziel) [inline cmd/trtl/main.go:651]:
```suggestion
	// Execute the Delete request
```

File: cmd/reissuer/build.sh [hunk 2]
```diff
@@ -80,6 +80,6 @@ if [[ $# -eq 1 ]]; then
 fi
 
 echo "your k8s cluster context is $KUBECTX"
-GOOS=linux GOARCH=amd64 go build -o $PWD/reissuer $DIR
+GOOS=linux GOARCH=amd64 go build -ldflags="-X 'github.com/trisacrypto/directory/pkg.GitVersion=$(git rev-parse --short HEAD)'" -o $PWD/reissuer $DIR
```
Comment (pdeziel) [inline cmd/reissuer/build.sh:83]: To get the correct Git tag into the binary?

**Review by pdeziel — APPROVED** [review]
This looks good to me, I just had one comment fix which is not from this PR but I noticed it when looking at this code again.

File: cmd/reissuer/build.sh (hunk 2 above)
Comment (bbengfort) [reply cmd/reissuer/build.sh:83]: Yep!

**Pull request merged**